learning_paths/migrations/0006_learningpathenrollment_enrolled_at_and_more.py

```diff
@@ -0,0 +1,66 @@
+# Generated by Django 4.2.19 on 2025-03-13 05:43
```

Comment:
Tiny nit: we can call this migration something more accurate, like `enrollments`.

Reply:
Yes! I have recreated this with the name `learning_path_enrollment_models`